### ISSUE TITLE
BUG: Can't open value in modal for tables without primary key

### ISSUE THREAD

**Issue by jboctor** [beekeeper-studio/beekeeper-studio#2586]
**Describe the bug**
Unable to edit (or view) table column in modal when faced with a table without a primary key.

**To Reproduce**
1. Go to a table without a primary key
2. Right click on a column
3. See the option to "Edit in modal" disabled

**Expected behavior**
Able to "Edit in modal" or at least be able to view in modal

**Screenshots / Gifs**
![image](https://github.com/user-attachments/assets/cb81caff-3420-4f6c-9204-eba96eebe898)

**Version Information (please complete the following information):**
 - OS: Debian Linux
 - App Version: 4.6.8
 - Database type and version: MySQL 8.1

**Comment by rathboma** [comment]
Agreed! We should def let you view in the modal, just not let you make changes
